### ISSUE TITLE
Re-POST request to another service

### ISSUE THREAD

**Issue by gdraganic** [balderdashy/skipper#117]
Is it possible to attach a file upload to another application to avoid buffering data (disk, S3, memory)? The idea is just to stream the data to another backend service if certain conditions are met. Simple piping the request does not work because of the body parser and I can not disable it because I need to validate some body params before I can resend the request. Any help would be highly appreciated.

**Comment by sailsbot** [comment]
Thanks for posting, @gdraganic.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
